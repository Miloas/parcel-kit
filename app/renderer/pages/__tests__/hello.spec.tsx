import * as React from 'react'
import * as Renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'

import { Hello } from '..'

test('page Hello', () => {
  const component = Renderer.create(
    <MemoryRouter><Hello /></MemoryRouter>,
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
