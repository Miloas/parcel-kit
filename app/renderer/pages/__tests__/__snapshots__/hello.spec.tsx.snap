// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page Hello 1`] = `
<div
  style={
    Object {
      "marginTop": 100,
      "textAlign": "center",
    }
  }
>
  <h1>
    Hello
  </h1>
  <h3>
    <a
      href="/next"
      onClick={[Function]}
    >
      next page
    </a>
  </h3>
</div>
`;
